mass_mailing_custom_unsubscribe/models/mail_mass_mailing.py

```diff
@@ -31,3 +33,42 @@ def update_opt_out(self, email, list_ids, value):
                 "action": action,
             })
         return super().update_opt_out(email, list_ids, value)
+
+    def update_opt_out_other(self, email, res_ids, value):
```

Comment:
What's the need for having this method separated from `update_opt_out` instead of handling opt out in any model directly there? :thinking:

Reply:
For having the possibility of extending it without needing to do all the `if` stuff, avoid calling super, etc. It's enough different concept for keeping it separated.